packages/form/core/form-control.ts

```diff
@@ -12,6 +12,7 @@ import type {
 	ValidatorRules,
 	ValidationHooks,
 } from '@astro-reactive/common';
+import { transformToValidatorRules } from '@astro-reactive/validator';
```

Comment:
Since we must use `resolver` together with `validator`, I thought about using `transformToValidatorRules` directly in the resolver function. The catch is that we can't build (with tsup) the `resolver` package because bundlers can't resolve astro files. My take is that if we don't care about building the package, it's a viable idea. On top of that, `resolver` only usable on server side and will be pruned anyway.

Other than that, there are 2 alternative solution I can think of currently:
1. User must use `transformValidatorRules` by themselves.
```ts
{ validationSchema: transformToValidatorRules(zodResolver(schema))}
``` 
This way, we can receive`ValidatorRules` from config.

2. We also can just put `resolver` as sub-package in `validator`  if we don't care about building.

Reply:
I think option 2 is worth trying. The only .astro file if I remember correctly in the `validator` package is `Validator.astro` which we just need for shipping client-side JS for validation on the browser. Other files are just `.ts` classes. I wonder if you can ignore `.astro` files when building?

Reply:
I found a way to ignore the `Validator.astro` file by exporting `core` too, rather than just exporting from index. Apparently, it caused error because the `Validator.astro` file got imported too when we import from `@astro-reactive/validator`. The solution was to import from `@astro-reactive/validator/core`. 

With this we can import `validator` in `resolver`